disperser/apiserver/server.go

```diff
@@ -824,6 +825,14 @@ func (s *DispersalServer) validateRequestAndGetBlob(ctx context.Context, req *pb
 		return nil, errors.New("number of custom_quorum_numbers must not exceed 256")
 	}
 
+	// validate every 32 bytes is a valid field element
+	_, err := rs.ToFrArray(data)
+	if err != nil {
+		s.logger.Error("failed to convert a 32bytes as a field element", "err", err)
+		s.metrics.HandleInvalidArgRequest("DisperseBlob")
```

Comment:
Since this validation is now called inside `validateRequestAndGetBlob`, this update to metric is not needed. The caller is already taking care of the errors in metric.